Makefile

```diff
@@ -56,20 +60,35 @@ tags:
 
 fetch_build:
 	curl -Lk -o $(ARCHIVE_PATH) $(BUILD_URL)/$(ARCHIVE_FILENAME)
+	if [ ! -z "$(DEPS_URL)" ]; then                                          \
+		set -e;                                                              \
+		FILENAME=$$(curl -k --list-only $(DEPS_URL)/repo/ | grep chroma-manager-integration | sed -e 's/.*\(chroma-manager-integration-[^/]*\)\/.*/\1/'); \
```

Comment:
`grep | sed` is an anti-pattern.  `sed` can grep on it's own using the `-n` switch and the `p` command as such:

`FILENAME=$$(curl -k --list-only $(DEPS_URL)/repo/ | sed -ne '/chroma-manager-integration/s/.*\(chroma-manager-integration-[^/]*\)\/.*/\1/p'); \`

Reply:
nice . thanks